docs/apache-airflow-providers-docker/decorators/docker.rst

```diff
@@ -0,0 +1,130 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+.. _howto/decorator:docker:
+
+Task Docker Decorator
+=====================
+
+Python callable wrapped within the ``@task.docker`` decorator with args are executed within
+the docker container.
+
+Parameters
+----------
+
+The following parameters are supported in Docker Task decorator.
```

Comment:
This list of parameters is long and duplicated from API documentation. It may be difficult to sync the two sources. Would it be possible to use autodoc to automatically pull this content from docstrings in Python files? Or maybe it’d be better to say the decorator takes all the same parameters as DockerOperator (except `task_id`), and link to the DockerOperator’s API documentation?

Reply:
yes i agree @uranusjr I tried using autodoc and it didn't work. I will try again and check or I will point to the DockerOperator's API documentation and mention about excluding task_id point

Reply:
i will see if i can fix this and raise as separate PR @uranusjr